helpers/build-url.js

```diff
@@ -0,0 +1,12 @@
+import qs from 'querystring';
+
+export default (url, imgServiceReqd) => {
+	const imageOptions = {
+		source: 'next',
+		fit: 'scale-down',
+		compression: 'best'
+	};
+
+	if (imgServiceReqd) url = `https:\/\/next-geebee.ft.com/image/v1/images/raw/${encodeURIComponent(url)}`;
+	return url += `?${qs.stringify(imageOptions)}&width=`;
+}
```

Comment:
to make this more generic, can also pass `width` to it, e.g. https://github.com/Financial-Times/n-handlebars/blob/master/src/helpers/resize.js